src/codec/meta/cid.rs

```diff
@@ -9,8 +10,6 @@ const CID_LENGTH: usize = 32;
 pub struct Cid([u8; CID_LENGTH]);
 
 impl Cid {
-    pub const IDENTITY: Cid = Cid([0u8; CID_LENGTH]);
```

Comment:
The identity CID here is a special case for things like empty files or non-content CIDs and a special value for the root of our merkle trees. Since there are these special values, and ones we'll want to regular operate against, I think it probably still makes sense to keep a const copy of this for when we need it. Like default() on the other structs, we should probably be explicit when we're using the identity CID as opposed to meaningful values that we generally want to accept.

There are some nice benefits to keep this as a constant CID such as not needing to do any form of construction when checking for the identity case but I'll admit its an almost trivial optimization that we wouldn't get much out of.